### PR TITLE
update ci to 0.7.0

### PR DESCRIPTION
**Pull request by pankore**
- CI use docker image 0.7.0
- Combine PR #25911 to fix CI error

### CODE REVIEW

**Comment by pankore** [comment]
Hi @andy31415 , it seems some platforms have build errors after updating the CI version. Should we skip this update for them?

File: src/platform/Ameba/NetworkCommissioningWiFiDriver.cpp [hunk 1]
```diff
@@ -228,8 +228,7 @@ void AmebaWiFiDriver::OnScanWiFiNetworkDone()
         return;
     }
 
-    rtw_scan_result_t * ScanResult = (rtw_scan_result *) pvPortMalloc(NumAP * sizeof(rtw_scan_result));
-    matter_get_scan_results(ScanResult, NumAP);
+    rtw_scan_result_t * ScanResult = matter_get_scan_results();
```
Comment (bzbarsky-apple) [inline src/platform/Ameba/NetworkCommissioningWiFiDriver.cpp:231]: This doesn't seem like "update CI" bits....

Comment (pankore) [reply src/platform/Ameba/NetworkCommissioningWiFiDriver.cpp:231]: Yes, this commit is actually to fix the CI error after update to 0.7.0.

**Review by bzbarsky-apple — APPROVED** [review]

**Comment by pankore** [comment]
Hi @andy31415 , please help to review this PR, many thanks!

**Review by andy31415 — APPROVED** [review]

**Pull request merged**